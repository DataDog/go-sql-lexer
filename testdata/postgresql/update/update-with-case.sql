UPDATE users SET status = CASE WHEN last_login < NOW() - INTERVAL '1 year' THEN 'inactive' ELSE status END;
