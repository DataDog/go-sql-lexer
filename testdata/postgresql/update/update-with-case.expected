[
    {
        "expected": "UPDATE users SET status = CASE WHEN last_login < NOW ( ) - INTERVAL ? THEN ? ELSE status END",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]